### PR TITLE
css-four-column-layout - Tim Silva

### PR DESCRIPTION
**Pull request by silva-tim**
Screen Shot 2023-05-22 at 6 18 40 PM

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          <div class="fourthColumn">
+            <img src="images/screen-3.png" alt="Service Filters" />
+          </div>
+          <div class="fourthColumn">
+            <img src="images/screen-4.png" alt="Stats and Mileage" />
+          </div>
+        </div>
+        <div class="row">
+          <div class="fourthColumn">
+            <img src="images/switch-icon.png" class="icon" />
```
Comment (shawnkost) [inline css-four-column-layout/index.html:36]: Needs an `alt` attribute

File: css-four-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            <h3>App Switching</h3>
+            <p>Toggle all your apps on and off.</p>
+          </div>
+          <div class="fourthColumn">
+            <img src="images/filter-icon.png" class="icon" />
```
Comment (shawnkost) [inline css-four-column-layout/index.html:41]: Needs an `alt` attribute

File: css-four-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            <h3>Trip Filtering</h3>
+            <p>Evaluates your trip requests.</p>
+          </div>
+          <div class="fourthColumn">
+            <img src="images/slider-icon.png" class="icon" />
```
Comment (shawnkost) [inline css-four-column-layout/index.html:46]: Needs an `alt` attribute

File: css-four-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            <h3>Service Filters</h3>
+            <p>Accepts only your ideal trips.</p>
+          </div>
+          <div class="fourthColumn">
+            <img src="images/bars-icon.png" class="icon" />
```
Comment (shawnkost) [inline css-four-column-layout/index.html:51]: Needs an `alt` attribute

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes here Tim! Just missing some `alt` attributes

File: css-four-column-layout/index.html (hunk 4 above)
Comment (silva-tim) [reply css-four-column-layout/index.html]: Added

**Pull request merged**